scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -0,0 +1,17 @@
+namespace <%= namespace %>.SDK {
+    using System.Collections.Generic;
+
+    public class ShopifyError {
+        public enum ErrorType {
+            HTTP, GraphQL
+        }
+
+        public readonly ErrorType Error;
+        public readonly string Reason;
```

Comment:
We should probably follow [Error](https://msdn.microsoft.com/en-us/library/t9zk6eay(v=vs.100).aspx), so `ShopifyError.description`